src/lib/Scenes/MyProfile/MyProfileHeaderMyCollectionAndSavedWorks.tests.tsx

```diff
@@ -136,14 +139,12 @@ describe("MyCollectionAndSavedWorks", () => {
           }),
         })
 
-        const text = extractText(wrapper.root)
-
-        expect(text).toContain("Princess")
-        expect(text).toContain("Member since 2012")
-        expect(text).toContain("Richest Collector! 💰")
-        expect(text).not.toContain("Guardian of the Galaxy")
-        expect(text).not.toContain("Atlantis")
-        expect(text).not.toContain("Marvel Universe")
+        expect(findByText("Princess")).toBeTruthy()
+        expect(findByText("Member since 2012")).toBeTruthy()
+        expect(findByText("Richest Collector! 💰")).toBeTruthy()
+        expect(findByText("Guardian of the Galaxy")).toEqual(Promise.resolve())
+        expect(findByText("Atlantis")).toEqual(Promise.resolve())
+        expect(findByText("Marvel Universe")).toEqual(Promise.resolve())
```

Comment:
These tests are confusing. Texts `toEqual(Promise.resolve())`?

Reply:
How is it working with `Promise.resolve()`? Would this work as well?

```suggestion
        expect(findByText("Guardian of the Galaxy")).not.toBeTruthy()
        expect(findByText("Atlantis")).not.toBeTruthy()
        expect(findByText("Marvel Universe")).not.toBeTruthy()
```

Reply:
Yes, it's returning an empty Promise for those strings. `toBeFalsy()` , `not.toBeTruthy()` and all other methods were failing.

Reply:
Yes @olerichter00, its failing 

    ` expect(received).not.toBeTruthy() `

    ` Received: {} `

Reply:
Well, `findByText` doesn't seem to be testing anything and always return an object. Let's try to use another way to check if the text is rendered (e.g. with a test ID).